test/runtime/Policies.go

```diff
@@ -109,12 +109,6 @@ var _ = Describe("RuntimeValidatedPolicyEnforcement", func() {
 			Expect(endPoints[helpers.Enabled]).To(Equal(1))
 		})
 
-		It("Handles missing required fields", func() {
-			By("Apply a policy with no endpointSelector without crashing")
-			_, err := vm.PolicyImportAndWait(vm.GetFullPath("no_endpointselector_policy.json"), helpers.HelperTimeout)
-			Expect(err).Should(BeNil())
```

Comment:
We could change this to `Expect(err).Should(Not(BeNil()))`

Reply:
We already have a unit test for this; there is no reason to have this in the Ginkgo tests.